### PR TITLE
Fix mini-css-extract-plugin types to work with Webpack 5

### PR DESCRIPTION
**Pull request by xarety**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Comment by xarety** [comment]
Hey, @RyanCavanaugh, seems like @typescript-bot hasn't reacted to this PR. Could you help, please!

File: types/mini-css-extract-plugin/index.d.ts [hunk 1]
```diff
@@ -6,18 +6,25 @@
...
+     * Webpack loader used always at the end of loaders list
+     */
     static loader: string;
 
     constructor(options?: MiniCssExtractPlugin.PluginOptions);
+
+    /**
+     * Apply the plugin
+     */
+    apply(compiler: Compiler): void;
```
Comment (JounQin) [inline types/mini-css-extract-plugin/index.d.ts:27]: Wondering why there is no `Plugin` interface any more.

**Review by JounQin — APPROVED** [review]

**Comment by xarety** [comment]
Ready to merge

File: types/mini-css-extract-plugin/index.d.ts (hunk 1 above)
Comment (peterblazejewicz) [reply types/mini-css-extract-plugin/index.d.ts:27]: There is, it's backported and exported as `WebpackPluginInstance` contract, compatible with 4.*/5.*,
but there is not strict requirement really to implement the interface here.

**Pull request merged**